test/integration/targets/assert/assert_quiet.out.quiet.stdout

```diff
@@ -0,0 +1,16 @@
+
+PLAY [localhost] ***************************************************************
+
+TASK [assert] ******************************************************************
+ok: [localhost] => (item=item_A)
+
+TASK [assert] ******************************************************************
+ok: [localhost] => (item=item_A) => {
+    "changed": false, 
+    "item": "item_A", 
+    "msg": "All assertions passed"
+}
+
+PLAY RECAP *********************************************************************
+localhost                  : ok=2    changed=0    unreachable=0    failed=0   
```

Comment:
There is `skipped=0` missing in this line (see for example the CI output [here](https://app.shippable.com/github/ansible/ansible/runs/108614/64/console)).

Reply:
Yep, my mistake, I captured output from 2.7.  Reran from 2.8.